Makefile

```diff
@@ -71,6 +72,13 @@ check-format:
 bundle-python:
```

Comment:
can you help change this as `bundle-python-mac` too? thanks!

Reply:
@pkong-ds updated